client/package.json

```diff
@@ -4,13 +4,18 @@
   "description": "Puppet VSCode extension",
   "version": "0.0.4",
   "publisher": "puppet",
-  "license": "Apache-2.0",
-  "icon":"images/Puppet-Logo-Amber-sm.png",
-  "repository":{
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "icon": "images/Puppet-Logo-Amber-sm.png",
+  "galleryBanner": {
+    "color": "#000000",
```

Comment:
This may be too 'dark', but then again i only work in black and sometimes very very dark grey